### PR TITLE
[SAIC-573] Exclude empty images from migration.

### PR DESCRIPTION
**Pull request by Settis**

### CODE REVIEW

File: cfglib.py [hunk 1]
```diff
@@ -171,6 +171,9 @@
     cfg.IntOpt('ssh_connection_attempts', default=3,
                help='Number of times CloudFerry will attempt to connect when '
                     'connecting to a new server via SSH.'),
+    cfg.BoolOpt('ignore_epmty_images',
```
Comment (roman-verchikov) [inline cfglib.py]: Option is misspelt, should be ignore_empty_images

File: cfglib.py [hunk 2]
```diff
@@ -171,6 +171,9 @@
...
+                help='Ignore images with size 0 and exclude they from '
```
Comment (roman-verchikov) [inline cfglib.py]: exclude them

File: configs/config.ini [hunk 3]
```diff
@@ -167,6 +167,8 @@ optimize_user_role_fetch = [True|False]
 # server via SSH.
 ssh_connection_attempts = 3
 
+# Ignore images with size 0 and exclude they from migration process
+ignore_epmty_images = False
```
Comment (roman-verchikov) [inline configs/config.ini]: Misspelt option, should be ignore_empty_images

**Comment by roman-verchikov** [comment]
Looks good other than few minor comments inline, please resolve

File: configs/config.ini [hunk 4]
```diff
@@ -167,6 +167,8 @@ optimize_user_role_fetch = [True|False]
 # server via SSH.
 ssh_connection_attempts = 3
 
+# Ignore images with size 0 and exclude they from migration process
```
Comment (roman-verchikov) [inline configs/config.ini]: "exclude them", not "exclude they"

**Comment by miarmak** [comment]
Looks good. +1

**Comment by roman-verchikov** [comment]
+1, can be merged.

**Pull request merged**